src/clr-angular/forms/datepicker/model/calendar-view.model.ts

```diff
@@ -80,12 +82,21 @@ export class CalendarViewModel {
     this.initializeFocusableDay();
   }
 
+  private isDateDisabled(date: DayModel) {
```

Comment:
we changed the `isDisabled` to `isExcluded` we could continue this and for:

```ts
isDateDisabled() => isDateExcluded()
disabledDates => excludedDates
```
And not confuse them with `disabled` dates as in the context of disabled input or options list.

Reply:
I'll update them for excluded.